builtin/verify-tag.c

```diff
@@ -12,12 +12,16 @@
 #include <signal.h>
 #include "parse-options.h"
 #include "gpg-interface.h"
+#include "ref-filter.h"
+#include "refs.h"
 
```

Comment:
You [mentioned]( https://github.com/lukpueh/git/pull/2/files#r79016961) that these two includes could go away if we have the the verify and format functionality in `tag.c`.
We actually have to keep `#include "ref-filter.h"` for `verify_ref_format` but I think we can strip the other one.